src/components/PopupCardDeleteVerify.js

```diff
@@ -29,4 +29,3 @@ export default class PopupCardDeleteVerify extends Popup {
     this._handleFormSubmit();
   }
 }
```

Comment:
```js

  close() {
    super.close();
  }
```

You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class.